docker/jenkins2-centos/contrib/jenkins/kube-slave-common.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/sh
+
+# AEROGEAR team update:
```

Comment:
*NOTE*: This file was copied here as temporary hack to seed right pod template. In future is going to be removed. Initial problem we had that kubernetes configuration was seeded. The right solution would be to disable pod templates generation in this file and have separate file that would create internal pod templates. After consultation with Luigi we decided to just copy this file for now and create ticket to do the right thing:

https://issues.jboss.org/browse/AGDIGGER-41 

I'm just worried that this comment would note that this file and change is desired way to do this. Copying shell scripts from base image is anti pattern and may lead to many different problems on later stage. 
Please note that we using latest tag for the base image so any change upstream may break this at any time (even after release). I personally hit so many issues with this approach when working on mysql and mongo docker images from upstream and I think that we should not go that route again :)

*Suggestion*: update comment to reflect that this is temporary solution.